tools/accuracy_checker/accuracy_checker/representation/image_inpainting.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from enum import Enum
+import numpy as np
+
+from .base_representation import BaseRepresentation
+from ..data_readers import BaseReader
+from ..preprocessor import Crop
+
+
+class GTLoader(Enum):
+    PILLOW = 0
+    OPENCV = 1
+
+
+class ImageInpaintingRepresentation(BaseRepresentation):
+    pass
+
+
+class ImageInpaintingAnnotation(ImageInpaintingRepresentation):
+    LOADERS = {
+        GTLoader.PILLOW: 'pillow_imread',
+        GTLoader.OPENCV: 'opencv_imread'
+    }
+
+    def __init__(self, identifier, path_to_hr, target_height, target_width, gt_loader=GTLoader.PILLOW):
+        """
+        Args:
+            identifier: object identifier (e.g. image name).
+            path_to_hr: path where height resolution image should be loaded from. The path is relative to data source.
+            gt_loader: back-end, used to load segmentation masks.
+        """
+
+        super().__init__(identifier)
+        self._image_path = path_to_hr
+        self.target_height = target_height
+        self.target_width = target_width
+        self._gt_loader = self.LOADERS.get(gt_loader)
+
+    @property
+    def value(self):
+        loader = BaseReader.provide(self._gt_loader, self.metadata['data_source'])
+        gt = loader.read(self._image_path)
+
+        h, w = gt.shape[:2]
+        if self.target_height is None:
+            self.target_height = h
+        if self.target_width is None:
+            self.target_width = w
+
+        gt = Crop.process_data(gt, int(self.target_height), int(self.target_width), None, False, True, {})
```

Comment:
not sure that it should be done here better to have postprocessing for it.
and also not sure that it should be done in case when target sizes are not provided